### ISSUE TITLE
lcms directory schema in TODO status errors on validation, expecting a TODO file/dir

### ISSUE THREAD

**Issue by NHPatterson** [hubmapconsortium/ingest-validation-tools#939]
Hi all,

I am preparing some LC-MS data for submission. Unfortunately this directory schema is in TODO status. We have the data organized but there is no way to validate it as it expects a TODO rather than just being pass-all.

**We get this telling error:**
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Required but missing: TODO.

See the entire validate_upload.py output below:
```
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_metadata/mqpar.xml.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/Carbamidomethyl (C)Sites.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/Oxidation (M)Sites.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/allPeptides.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/evidence.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/matchedFeatures.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/modificationSpecificPeptides.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/msms.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/msmsScans.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/mzRange.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/parameters.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/peptides.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/proteinGroups.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/summary.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/ID_search_results/tables.pdf.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/PTXQC_report_template.Rmd.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/report_v1.0.9_VAN0006-LK-4.html.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/report_v1.0.9_VAN0006-LK-4.pdf.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/report_v1.0.9_VAN0006-LK-4.yaml.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/report_v1.0.9_VAN0006-LK-4_filename_sort.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/QC_results/report_v1.0.9_VAN0006-LK-4_heatmap.txt.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Not allowed: Proteomics/raw_data/VAN0006-LK-4.raw.
Metadata TSV Errors: U:\TUDelftShare\BIOMIC-DR2-LCMS\lcms-metadata.tsv (as lcms): External: row 2, data U:\TUDelftShare\BIOMIC-DR2-LCMS\VAN0006-LK-4: Required but missing: TODO.
````

Let us know how we can help,
Heath
Vanderbilt TMC

**Comment by mccalluc** [comment]
@icaoberg / @NHPatterson -- It's not going to work. There are multiple syntax errors. I'll detail them below.

**Comment by NHPatterson** [comment]
Hi all, it seems this has been fixed with #989. I'm able to validate lcms data.
